product/reports/670_Performance by Asset Type - Middleware Servers/110_JVM Heap Usage - daily over last week.yaml

```diff
@@ -0,0 +1,71 @@
+--- 
+where_clause: 
+dims: 
+created_on: 2016-05-23 14:53:10.822035 Z
+reserved: 
+title: "JVM Heap Usage - daily averages for last week"
+conditions: 
+updated_on: 2016-05-23 00:35:33.732482 Z
+order: Descending
+graph: 
+menu_name: "JVM Heap Usage - daily over last week"
+rpt_group: Custom
+priority: 
+col_order: 
+- id
+- name
+- start_time
+- mw_heap_used_min
+- mw_heap_used_avg
+- mw_heap_used_max
+- mw_heap_committed_min
+- mw_heap_committed_avg
+- mw_heap_committed_max
+- mw_heap_max_min
+- mw_heap_max_avg
+- mw_heap_max_max
+timeline: 
+id: 99
+file_mtime: 
+categories: 
+rpt_type: Custom
+filename: 
+db_options:
+  :rpt_type: middleware
+  :options:
+    :interval: daily
+    :start_offset: 604800
+    :end_offset: 0
+include: {}
+
+db: MiddlewareServerPerformance
+cols: 
+- id
+- name
+- start_time
+- mw_heap_used_min
+- mw_heap_used_avg
+- mw_heap_used_max
+- mw_heap_committed_min
+- mw_heap_committed_avg
+- mw_heap_committed_max
+- mw_heap_max_min
+- mw_heap_max_avg
+- mw_heap_max_max
```

Comment:
can you please explain what is the meaning of Max-max? not really clear to me from a glance